lib/chemistrykit/cli/cli.rb

```diff
@@ -126,7 +126,8 @@ def rspec_config(config)
         end
 
         ::RSpec.configure do |c|
-          c.capture_log_messages
+          log = Logging.logger['test steps']
+          c.capture_log_messages          
```

Comment:
does it work with this line?  This was supposed to output to a file at the end of the test, but you're doing that explicitly by adding an Appender now.